test/tests/autotranslation.js

```diff
@@ -18,11 +18,16 @@ var BeakerPageObject = require('./beaker.po.js');
 var path = require('path');
 
 describe('autotranslation', function() {
-  beforeEach(function() {
+  beforeEach(function(done) {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000; // Slow initialization cells, CI box might need a long time to initialize these
     beakerPO = new BeakerPageObject();
-    browser.get(beakerPO.baseURL);
-    browser.waitForAngular();
+    browser.get(beakerPO.baseURL).then(
+        function(){done();},
+        function(reason){ console.log('error occurred while browser load page : ' + reason);}
+    );
+    browser.waitForAngular().then(function() {
+      done();
```

Comment:
So what happens when we call done() the second time? I thought the way this worked is that when the done is called, the tests will start. Seems a bit weird to call done twice.
I thought that we want to get the baseURL 'then' wait for angular and 'then' call the done() callback:

    browser.get(beakerPO.baseURL)
      .then(browser.waitForAngular)
      .then(done);